sales.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <link rel="stylesheet" type="text/css" href="style.css">
+  <meta charset="UTF-8">
+  <title>Sales</title>
+  <nav>
+    <ul id="navigation">
+      <li><a href='file:///Users/codyhulsey/projects/cookie_stand_201/mainpage.html'>HOME</a></li>
+      <li><a href="file:///Users/codyhulsey/projects/cookie_stand_201/index.html">SALES</a></li>
+      <li>
+      <a href="file:///Users/codyhulsey/projects/cookie_stand_201/styleguide.html">STYLE GUIDE</a></li>
+    </ul>
+  </nav>
+</head>
+<body>
+  <form id="myForm">
+    <fieldset>
+      <legend>Create your own store</legend>
+
+      <label for="storeName">Store Name</label>
+      <input type="text" id='storeName'></input>
+  </fieldset>
+
+  <fieldset>
+    <legend>How much traffic does this store experience</legend>
+      <label for='min'>
+        Minimum Customers
+      <input type="number" id='min'></input>
+      </label>
+
+      <label for="max">Maximum Customers</label>
+      <input type="number" id='max'></input>
+    </fieldset>
+
+    <fieldset>
+      <legend>On average, how many customers purchase something every hour</legend>
+      <label for="avg">Average Customers Per Hour</label>
+      <input type="text" id='avg'></input>
+    </fieldset>
+
+    <fieldset>
+      <legend>Submit</legend>
+      <input type='submit' value='Submit' id='submit'</input>
+    </fieldset>
+  </form>
+  <main>
```

Comment:
The form probably belongs inside of the main tag as well, given the form is part of the main content of the page.